app/views/home/businesses.html.erb

```diff
@@ -97,41 +97,41 @@
   <div class="container-lg">
     <h2 class="alt-h2 text-center">You're in good company</h2>
 
-    <div class="text-center my-6 py-6 text-gray-light" style="">
-      [QUOTE FROM A COMPANY ON WHY CONTRIBUTING TO OPEN SOURCE IS IMPORTANT]
-    </p>
-
-    <div class="text-center my-6 py-6 alt-h2 text-gray-light" style="">
-      [COMPANY LOGOS HERE]
-    </div>
-
     <div class="clearfix mt-6 pt-6 mb-4 border-top">
       <div class="col-sm-6 float-left mb-4 mb-md-0 pr-sm-3 pr-lg-4">
         <div>
           <div class="d-table-cell v-align-top pr-3">
-            <img src="https://api.adorable.io/avatars/285/ossfriday.png" alt="" width="50" height="50" style="border-radius:25px;">
+            <img src="https://avatars1.githubusercontent.com/u/691109?v=3&s=100" alt="" width="50" height="50" style="border-radius:25px;">
           </div>
           <blockquote class="d-table-cell v-align-top text-gray">
-            <p class="alt-mono-font alt-text-small">“Quote that talks about how some amazing company invests in open source.”</p>
-            <footer class="alt-text-small">Jane Smith, Tech Giant</footer>
+            <p class="alt-mono-font alt-text-small">Open source aligns with our mission to make the world more open and connected. Working with the community gives us an opportunity to share our work and the challenges we’re aiming to solve, and encourages talented people to contribute their ideas and effort to move the world forward.</p>
+            <footer class="alt-text-small">Christine Abernathy (<a href="https://github.com/caabernathy">@caabernathy</a>), Open Source at <a href="https://github.com/facebook">Facebook</a></footer>
           </blockquote>
         </div>
       </div>
       <div class="col-sm-6 float-left pl-sm-3 pl-lg-4">
         <div>
           <div class="d-table-cell v-align-top pr-3">
-            <img src="https://api.adorable.io/avatars/285/ossfriday2.png" alt="" width="50" height="50" style="border-radius:25px;">
+            <img src="https://avatars0.githubusercontent.com/u/1112?v=3&s=100" alt="" width="50" height="50" style="border-radius:25px;">
           </div>
           <blockquote class="d-table-cell v-align-top text-gray">
-            <p class="alt-mono-font alt-text-small">“Another quote that talks about how Open Source Friday saved my business and my dog.”</p>
-            <footer class="alt-text-small">Erica McGitterson, Bootstraps R Us</footer>
+            <p class="alt-mono-font alt-text-small">Contributing to open source projects continues to be one of the most popular 20% projects for Google engineers. Open source plays such an important role in so much of what we do, so we try to make <a href="https://opensource.google.com/docs/patching/">the process</a> as easy as possible.</p>
+            <footer class="alt-text-small">Will Norris (<a href="https://github.com/willnorris">@willnorris</a>), Open Source at <a href="https://github.com/google">Google</a></footer>
```

Comment:
@willnorris Just a heads up that your quote is being used here. Let me know either here or at mikemcquaid@github.com if you have any problem with that. Ignore this if you're 👍. Thanks!